### DISCUSSION TITLE
It would be good if Executors did not at [end of text] or \n> at the end of the inference

### DISCUSSION THREAD

**Discussion by clovisribeiro** [SciSharp/LLamaSharp#581] [Ideas]
It would be great if LLamaInteractExecutor and LLamaInstructExecutors did not add extra stuff at the end of the inference's result. (They currently add "[end of text]" and "\n> " respectively. I believe many applications using those executors will be doing something else with the result, so having the plain text without any extra stuff is better.  
I guess the demos showing how to use those executors could add the extra stuff.

**Comment by patrick-hovsepian** [comment]
Take a look at the `KeywordTextOutputStreamTransformer`, you might able to use this to knock off those special tokens. I'm working on a PR to help make this more generic which is currently open.
